### PR TITLE
Fixbug/polls

### PR DESCRIPTION
**Pull request by FranzLuepke**
- Wake up polls added.
- Late checkout polls added.

### CODE REVIEW

File: apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data: WakeUpEntity,
+    private dialogRef: MatDialogRef<ModalWakeUpQualifyComponent>,
+    public pollsService: PollsService,
+    private guestService: GuestService,
+    private wakeUpService: WakeUpService,
+  ) {}
+
+  ngOnInit() {
+    this.guestService.$hotel.pipe(first()).subscribe((hotel) => {
```
Comment (ruedagato) [inline apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts:34]: use store

File: apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      this.pollsService.getPoll(hotel.uid, this.pollServiceConstant.wakeUp).subscribe((data) => {
+        this.service = this.pollServiceConstant.wakeUp;
+        this.polls = { [this.pollServiceConstant.wakeUp]: data };
```
Comment (ruedagato) [inline apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts:37]: do you need many polls or only one ?

File: apps/guest/src/home/home.module.ts [hunk 3]
```diff
@@ -60,6 +62,11 @@ import { ModalPromoComponent } from './components/modal-promo/modal-promo.compon
     TranslateModule,
     DynamicTranslateModule,
   ],
-  entryComponents: [ModalQualifyComponent, ModalBookingQualifyComponent, ModalOrdersQuialifyComponent],
+  entryComponents: [
```
Comment (ruedagato) [inline apps/guest/src/home/home.module.ts:65]: delete entryComponentes it is not necessary

File: apps/guest/src/home/pages/welcome/welcome.component.ts [hunk 4]
```diff
@@ -165,6 +171,34 @@ export class WelcomeComponent implements OnInit {
       });
   }
 
+  qualifyWakeUp() {
+    this.$guest
```
Comment (ruedagato) [inline apps/guest/src/home/pages/welcome/welcome.component.ts:175]: use store

File: apps/guest/src/home/pages/welcome/welcome.component.ts [hunk 5]
```diff
@@ -165,6 +171,34 @@ export class WelcomeComponent implements OnInit {
...
+      .pipe(
+        take(1),
+        switchMap((guest) => this.wakeService.getWake(guest!.uid)),
+        map((wakes) => wakes.filter((wake) => wake.complete === true && wake.qualification == null)),
+        filter((data) => data.length > 0),
+      )
+      .subscribe((data) => {
+        data.forEach((item) => {
+          if (item) {
+            this.$guest.subscribe((guest) => {
```
Comment (ruedagato) [inline apps/guest/src/home/pages/welcome/welcome.component.ts:185]: use store

File: apps/guest/src/home/pages/welcome/welcome.component.ts [hunk 6]
```diff
@@ -165,6 +171,34 @@ export class WelcomeComponent implements OnInit {
...
+    this.$guest
+      .pipe(
+        take(1),
+        switchMap((guest) => this.wakeService.getWake(guest!.uid)),
+        map((wakes) => wakes.filter((wake) => wake.complete === true && wake.qualification == null)),
+        filter((data) => data.length > 0),
+      )
+      .subscribe((data) => {
+        data.forEach((item) => {
+          if (item) {
```
[the remark this answers is not in the document]
Comment (FranzLuepke) [reply apps/guest/src/home/pages/welcome/welcome.component.ts]: No, is not. Erased.

File: apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts (hunk 2 above)
Comment (FranzLuepke) [reply apps/guest/src/home/components/modal-wake-up-qualify/modal-wake-up-qualify.component.ts]: Just one. Corrected.

**Review by ruedagato — APPROVED** [review]

**Pull request merged**